### ISSUE TITLE
Missing site navigation

### ISSUE THREAD

**Issue by claireristow** [Lullabot/lb-ebook-poc#36]
The lack of navigation to move around the e-book site is considered an accessibility issue (see [WCAG 2.2 Level AA: Multiple Ways](https://www.w3.org/WAI/WCAG22/Understanding/multiple-ways.html)).

Consider adding navigation at the top or even making the Lullabot logo link to the homepage of the ebook site rather than lullabot.com. When you're in an e-book, a link back to the table of contents for that ebook would also be really useful.

**Comment by huntermacd** [comment]
Resolved by #20.
